.cspellignore

```diff
@@ -21,3 +21,5 @@ subpage
 Traefik
 typesafe
 Undeprecate
+blocklist
+cbdf
```

Comment:
This is required because of the id here: https://github.com/vivid-planet/comet/blob/efd0797478b07d1f2140b2852b8e32db6dac10c0/docs/docs/4-guides/1-how-to-build-a-custom-client/index.md?plain=1#L607